src/convertToObject.js

```diff
@@ -12,7 +12,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const splitSouresSting = sourceString.split('\n');
```

Comment:
Theoretically, styles can be written in one line, so splitting them by '\n' - is not the best idea.
But styles will always end with ';' so you should use it for splitting.

Additionally - this will allow you not to perform additional slice in the end.